src/client/components/app-core/form.css

```diff
@@ -1,14 +1,10 @@
-[type="text"],
-[type="email"],
-[type="password"] {
+.form-control {
```

Comment:
`form-input`? or is `form-control` better because it also includes `textarea` and prevents the confusion that it only applies to `input`s?